### ISSUE TITLE
mint test suite error: test_copy_object_with_metadata

### ISSUE THREAD

**Issue by zaibon** [threefoldtech/minio#60]
One of the test from the minio/mint test suite failed: https://gist.github.com/chrisvdg/80a1a33f122046575405c022982854a6#file-monio_mint_full_result-json-L25

**Comment by ramezsaeed** [comment]
Test code
- https://github.com/minio/minio-py/commit/b9a1a70098691c6df4e0c14d0dd0b6dafd629e4f#diff-123f6df576c99ae259903f693e5bc6a4

**Comment by muhamadazmy** [comment]
Question: is this one of our tests, or from minio itself ?

**Comment by zaibon** [comment]
mint is the official minio compatibility test suite against s3 protocol. It maintained by the minio team

**Comment by muhamadazmy** [comment]
@ramezsaeed I need more details about this test and where does it run. It's not part of minio repo, hence u might be running incompatible version of the test since our code base is way out of date with the mainstream minio.

**Comment by zaibon** [comment]
@muhamadazmy : check https://github.com/minio/mint/

**Comment by muhamadazmy** [comment]
Running the tests on the vanilla minio at the commit where we forked also fails. This means this test is failing on a probably new feature.

To rebase our code base it's a lot of changes, since minio code base has evolved dramatically since we forked from it. As a solution for this issue we can do ONE of the following:

- Run mint tests from before they introduced this test/feature
- Reimplement zerostor integration in minio using minio recent code base. While at it also fix meta data limitations and other hacks, this is of course is not a quick solution
